backends-velox/src/test/scala/io/glutenproject/execution/VeloxStringFunctionsSuite.scala

```diff
@@ -154,6 +154,13 @@ class VeloxStringFunctionsSuite extends WholeStageTransformerSuite {
       s"from lineitem limit $LENGTH") { checkOperatorMatch[ProjectExecTransformer] }
   }
 
+  test("sha1") {
+    runQueryAndCompare(s"select l_orderkey, sha1(l_comment) " +
+      s"from lineitem limit $LENGTH") { checkOperatorMatch[ProjectExecTransformer] }
```

Comment:
It looks it execution goes into fallback path due to validation failed, which makes the test fail:
```
Validation failed for expression in ProjectRel due to:Found incompatible return types for 'sha1' (VARBINARY vs. VARCHAR) for input types (VARBINARY).
```

Reply:
Failure is due to the need to merge  https://github.com/oap-project/velox/pull/132 before this.

Reply:
https://github.com/oap-project/velox/pull/132 has been merged